app/code/core/Mage/Admin/Model/User.php

```diff
@@ -526,7 +526,7 @@ public function getPasskeyCreateArgs(): stdClass
     public function getPasskeyGetArgs(): stdClass
     {
         if (!$this->getPasskeyCredentialIdHash() || !$this->getPasskeyPublicKey()) {
-            Mage::throwException(Mage::helper('adminhtml')->__('Passkey is not enabled for this account.'));
+            Mage::throwException(Mage::helper('adminhtml')->__('You did not sign in correctly or your account is temporarily disabled.'));
```

Comment:
I also thought this was important. Although this error message is very unclear, I think it's better to be safer